canvas/circle_test.go

```diff
@@ -22,3 +24,29 @@ func TestCircle_FillColor(t *testing.T) {
 
 	assert.Equal(t, c, circle.FillColor)
 }
+
+func TestCircleReSize(t *testing.T) {
```

Comment:
It's really picky of me, sorry, but can you use the same casing as the method under test?
In other tests we use the format TestCircle_Resize - but I cannot find the doc where it is specified - maybe it's GoLand that does that automatically...

Reply:
Ah sure. The tests  aren't consistently like that outside of canvas/, seems to vary between the two styles depending on author.

I'll of course swap these two over

Reply:
Missed the S in `ReSize` will fix in a few